cgmes-gl-server/pom.xml

```diff
@@ -163,5 +163,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
we don't have this dependency in other microservices, it's best to remove it (it is transitively pulled by springdoc-openapi-ui)